spec/operators-spec.coffee

```diff
@@ -390,7 +390,7 @@ describe "Operators", ->
 
     describe "with multiple cursors", ->
       it "deletes each selection", ->
-        editor.setText("abcd\n1234\nABCD")
+        editor.setText("abcd\n1234\nABCD\n")
```

Comment:
the missing `\n` above was triggering another bug uncovered by this PR's fix: bug #622 is quite unrelated to this PR but the bug fixed by this PR was behaving just like we expect `de` to behave at the end of the file. With the `\n` above, the test still tests the same expected behaviour, but without triggering #622.